CMake/macros.cmake

```diff
@@ -249,8 +244,7 @@ function(SIMPLE_RUN_AND_CHECK base_name base_dir input_file procs threads check_
 
   # add run (task 1)
   set (test_added false)
-  set (test_labels "")
-  RUN_QMC_APP(${full_name} ${base_dir} ${procs} ${threads} test_added test_labels ${input_file})
+  RUN_QMC_APP(${full_name} ${base_dir} ${procs} ${threads} test_added "integration" ${input_file})
```

Comment:
test_labels is used on line 275 and needs to be returned here.

Reply:
I just invoke ADD_TEST_LABELS

Reply:
The dependent tests do not have the same name and are not processed the same by the labeling script.  They should just share the labels from the first call.

Reply:
The flow of test label information was done intentionally.

Reply:
I think so.  What labels does the script report for these when you run it?